### ISSUE TITLE
Unable to clear Litespeed Cache when logged-in as Editor

### ISSUE THREAD

**Issue by chandra10207** [litespeedtech/lscache_wp#303]
HI,

I can't see any options to clear Cache if I was logged-in with the Editor roles and added or updated some pages.

**Comment by qtwrk** [comment]
Hi,

yes,  I have a small addon for this 

https://github.com/qtwrk/lscwp-addon

create a directory in /wp-content/plugins/ , name `lscwp-addon`  , then create a php file inside of it , name `lscwp-addon.php` ,  then put this [code](https://raw.githubusercontent.com/qtwrk/lscwp-addon/main/lscwp-addon/lscwp-addon.php) into that file 

then go to plugin page to activate it.

**Comment by chandra10207** [comment]
@qtwrk,

Thankx for sharing but I can't save addons setting page If I have activated Porto Theme with Porto Theme Functionality Plugin activated.

However, It works with default twenty twenty theme.

Is there any hooks that I can use on child theme without using extra addon plugin setting, which can enable all litespeed purge options to the user with role editor.

**Comment by qtwrk** [comment]
yeah well , when I wrote that thing , I was using the WP's default theme to play with  : )

the idea behind that addon was:

1. create an admin bar item ,you can check it [here](https://developer.wordpress.org/reference/classes/wp_admin_bar/add_node/)

2. then link it to a purge action , you can check it [here](https://docs.litespeedtech.com/lscache/lscwp/api/#purge)
